README.md

```diff
@@ -1,95 +1,26 @@
-# Ember Singularity
+# ember-singularity
 
-Ember Singularity integrates a [Unified Event Handler](https://github.com/trentmwillis/ember-singularity/blob/master/app/services/unified-event-handler.js)
```

Comment:
We should bring back the real README (looks like this is the default addon README and looses the custom info that existed)..

Reply:
Thanks for catching, reset the entire file.